src/huggingface_hub/hf_api.py

```diff
@@ -1457,6 +1457,178 @@ def __init__(self, **kwargs) -> None:
         self.__dict__.update(**kwargs)
 
 
+@dataclass
+class PaperAuthor:
```

Comment:
I would suggest removing `PaperAuthor` if we only need the authors names. we can represent authors as a simple list of names `List[str]`.